include/NeoFOAM/finiteVolume/cellCentred/boundary/surface/calculated.hpp

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2023 NeoFOAM authors
+
+#pragma once
+
+#include "Kokkos_Core.hpp"
+
+#include "NeoFOAM/core.hpp"
```

Comment:
I think this might not work and we might have to replace this include with (untested)
```suggestion
#include "NeoFOAM/core/Dictionary.hpp"
```